.github/workflows/ci.yaml

```diff
@@ -0,0 +1,23 @@
+name: Java CI
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    if: "!contains(github.event.head_commit.message, 'skip ci')"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup JDK 16
+        uses: actions/setup-java@v1
+        with:
+          java-version: 16
+      - name: Test
+        run: mvn --batch-mode --update-snapshots verify -s .mvn/settings.xml
+        env:
+          MAVEN_TOKEN: ${{ secrets.GH_TOKEN }}
+          MAVEN_USER: ${{ secrets.GIT_AUTHOR_NAME }}
```

Comment:
```suggestion
          MAVEN_USER: ${{ secrets.GIT_AUTHOR_NAME }}
          
```

Reply:
EOL

Reply:
I think I'm missing something. Is it switching the EOL or an EOF blank line?